magicbook/stylesheets/components/typography.scss

```diff
@@ -12,6 +12,8 @@ p {
   margin-top: 10pt;
   margin-bottom: 10pt;
   font-weight: $font-book;
+  orphans: 2;
```

Comment:
@jasongao97 I noticed you added these, do we know how well they work? Should we be investigating "prince for books" as well?

Reply:
> @jasongao97 I noticed you added these, do we know how well they work? Should we be investigating "prince for books" as well?

Yes! They effectively prevent orphan lines at the bottom or top of a page. Prince book is for better line-break and spread-balancing which I will investigate today.